mautic/mautic
# Add styling for selected button

[//]: # ( Please answer the following questions: )

| Q  | A
| --- | ---
| Bug fix? | x
| New feature? | 
| Related user documentation PR URL | 
| Related developer documentation PR URL | 
| Issues addressed (#s or URLs) | #2963 
| BC breaks? | 
| Deprecations? | 

[//]: # ( Note that all new features should have a related user and/or developer documentation PR in their respective repositories. )

[//]: # ( Required: )
#### Description:
Small CSS fix to stop the selected button jumping by 5px

[//]: # ( As applicable: )
#### Steps to reproduce the bug:
1. Create page or email, select theme
2. Notice that the button jumps up by 5px when marked as selected

#### Steps to test this PR:
1. Apply PR
2. Recompile CSS files from Less
3. Clear cache
4. Create landing page or email
5. Notice that the button no longer jumps when selected.
